sources/packages/api/src/services/disbursement-schedule-service/disbursement-schedule-service.ts

```diff
@@ -305,4 +329,127 @@ export class DisbursementScheduleService extends RecordDataModelService<Disburse
     coeQuery.orderBy("disbursementSchedule.coeStatus");
     return coeQuery.getMany();
   }
+
+  /**
+   * Returns Disbursement and application details for COE Application details.
+   * @param disbursementScheduleId
+   * @returns Disbursement and Application details.
```

Comment:
pls update the comment

Reply:
comment updated

Reply:
i think missed this. `locationId` is not added